python/paddle/fluid/dygraph/nn.py

```diff
@@ -1388,7 +1378,7 @@ def forward(self, input):
             "SavedVariance": [saved_variance]
         }
         if reserve_space is not None:
```

Comment:
Can remove since reserve_space always not None.

Reply:
It will be removed in the next PR along with the previous review. Thank you.